src/plasma/plasma_store.cc

```diff
@@ -48,27 +44,11 @@ void dlfree(void *);
 size_t dlmalloc_set_footprint_limit(size_t bytes);
 }
 
-/** Contains all information that is associated with a Plasma store client. */
-struct Client {
```

Comment:
Why remove the client struct? Don't we want to keep around the struct (even if it's only wrapping a file descriptor at the moment)?